src/Riok.Mapperly/Configuration/EnumMappingConfiguration.cs

```diff
@@ -11,6 +11,7 @@ public record EnumMappingConfiguration(
     IReadOnlyCollection<IFieldSymbol> IgnoredTargetMembers,
     IReadOnlyCollection<EnumValueMappingConfiguration> ExplicitMappings,
     RequiredMappingStrategy RequiredMappingStrategy,
+    RequiredMappingStrategy RequiredEnumMappingStrategy,
```

Comment:
I don't think this member is needed, is it? The existing `RequiredMappingStrategy` can be used.

Reply:
When I remove this from EnumMappingConfiguration, MapperConfigurationReader & EnumMappingDiagnosticReporter both these test don't work anymore:
- MapperAttributeRequiredEnumMappingSourceAndRequiredMappingNoneWithUnmappedMember
- MapperAttributeRequiredEnumMappingTargetAndRequiredMappingNoneWithUnmappedMember

In these tests I needed to change `ctx.Configuration.Enum.RequiredEnumMappingStrategy` tp `ctx.Configuration.Enum.RequiredMappingStrategy`.

Reply:
If I change it, all tests pass, find my changes I quickly tried in this [patch](https://github.com/user-attachments/files/17830203/patch.patch). Why don't the tests work?

Reply:
I'm not sure what I did wrong, it indeed works as expected now. Thanks!